tests/release/pipelines/rh_push_to_external_registry.go

```diff
@@ -298,7 +299,7 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 					return fmt.Errorf(errMsg)
 				}
 				return nil
-			}, releaseConst.ReleasePipelineRunCompletionTimeout, constants.PipelineRunPollingInterval).Should(Succeed())
+			}, 100 * time.Minute, constants.PipelineRunPollingInterval).Should(Succeed())
```

Comment:
If we aren't using the constant here, why have it? If the pipelines need different timeouts (IIRC the fbc one is 60 mins) maybe we just don't have a value in const.go and instead have one timeout value per test file

Reply:
I ever met timeout  issue with small cluster type.  Updated it back.  Thanks